site/app/controllers/student/SubmissionController.php

```diff
@@ -486,7 +486,7 @@ public function ajaxBulkUpload($gradeable_id) {
                     "is_qr"     => false
                 ];
 
-                $bulk_upload_job  = rawurlencode("/var/local/submitty/daemon_job_queue/bulk_upload_" . $uploaded_file["name"][$i] . ".json");
+                $bulk_upload_job  = "/var/local/submitty/daemon_job_queue/bulk_upload_" . rawurlencode($uploaded_file["name"][$i]) . ".json";
```

Comment:
This was added in #10048, I believe that the rawurlencode for the full path adds characters, and the path is no longer valid. Changing it to just `rawurlencode` the file name is a all that was needed in the first place, as the rest of the path are already valid characters.